### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by vincent-olivert-riera** [open-source-labs/vno#98]
I just followed the installation instructions to the letter and this is the problem:

```
$ deno -V
deno 1.8.1

$ deno install --allow-net --unstable https://deno.land/x/vno/install/vno.ts
Download https://deno.land/x/vno/install/vno.ts
Warning Implicitly using latest version (v1.2.1) for http://deno.land/x/vno/install/vno.ts
[snip]
Check https://deno.land/x/vno/install/vno.ts
error: Uncaught SyntaxError: The requested module 'https://cdn.esm.sh/v15/_node_fs.js' does not provide an export named 'default'
import __fs$ from "https://cdn.esm.sh/v15/_node_fs.js";
       ~~~~~
    at <anonymous> (https://raw.githubusercontent.com/crewdevio/vue-deno-compiler/master/src/postcss/postcss.js:6:8)
```

**Comment by buttercubz** [comment]
please see #96

**Comment by vincent-olivert-riera** [comment]
Closed as duplicate of #96
